### ISSUE TITLE
Custom status

### ISSUE THREAD

**Issue by kaepora** [cryptocat/cryptocat#74]
Someone suggested users should be able to customize their status ("away", etc., and leave a status message.) Seems like a good idea.

If this is implemented the [documentation](https://crypto.cat/help.html) should make sure to note how custom statuses are not encrypted (being general metadata.)

**Comment by TumM0reNcE5SsIti** [comment]
Status messages, statuses and anything more than online/offline is adding overhead to the server that isn't necessary for this application. Keeping it lightweight, sending/receiving relevant data (messages) should be the focus of Cryptocat.

I don't support this suggestion as it would require more information processing server/client side than is necessary, and is purely cosmetic.
We need to know if our users are online/offline. This isn't Skype.

**Comment by kaepora** [comment]
Sorry, no. I'd like to offer users some basic abilities like that.

**Comment by TumM0reNcE5SsIti** [comment]
Fair enough. I just think that a status of "Online" or "Offline" is better than complicating the application and it's calls.
+1 for standing your ground, sir!

**Comment by andersbateva** [comment]
@kaepora , just "AFK" should be enough to indicate the reply will not be instantaneous. A status message is something less necessary.
